src/SIM/SIMbase.C

```diff
@@ -1285,7 +1285,7 @@ bool SIMbase::solutionNorms (const TimeDomain& time,
           if (ssol[k].empty())
             norm->getProjection(k).clear();
           else
-            pch->extractNodeVec(ssol[k],norm->getProjection(k),nCmp);
+            this->extractPatchSolution(ssol[k],norm->getProjection(k),lp-1,nCmp,1);
```

Comment:
yes, your long-term memory does not fail you. short term however.. ;) i did ask on skype about the breakage. in particular the local nodes.